src/langs/en/auth.json

```
@@ -15,5 +15,7 @@
   "SIGN_IN_SUCCESS_TITLE": "Welcome back!",
   "SIGN_UP_SUCCESS_TITLE": "Welcome!",
   "SIGN_IN_SUCCESS_TEXT": "Please check your inbox <bold>{{email}}</bold> to retrieve your personal login link to access Graasp. This can take several minutes.",
-  "SIGN_IN_SUCCESS_EMAIL_PROBLEM": "If you did not receive any email, check your spam. If you used an institutional email (ie. school, company), it might have been blocked and will have to wait until the email is release by your spam system."
+  "SIGN_IN_SUCCESS_EMAIL_PROBLEM": "If you did not receive any email, check your spam. If you used an institutional email (ie. school, company), it might have been blocked and will have to wait until the email is release by your spam system.",
```

It's not from your code, but could you fix this typo? Thanks!

```suggestion
"SIGN_IN_SUCCESS_EMAIL_PROBLEM": "If you did not receive any email, check your spam. If you used an institutional email (ie. school, company), it might have been blocked and will have to wait until the email is released by your spam system.",
```